ygnmi/ygnmi.go

```diff
@@ -127,8 +127,9 @@ func (v *Value[T]) String() string {
 
 // Client is used to perform gNMI requests.
 type Client struct {
-	gnmiC  gpb.GNMIClient
-	target string
+	gnmiC               gpb.GNMIClient
+	target              string
+	requestDumpLogLevel log.Level
```

Comment:
"requestLogLevel" may be sufficiently descriptive?

Reply:
+1